django_auth_ldap/backend.py

```diff
@@ -1002,6 +1001,14 @@ def __init__(self, prefix='AUTH_LDAP_', defaults={}):
             value = getattr(django.conf.settings, prefix + name, default)
             setattr(self, name, value)
 
+        # Compatibility with old caching settings: if CACHE_TIMEOUT isn't changed and CACHE_GROUPS is True
+        if getattr(django.conf.settings, prefix + 'CACHE_GROUPS', defaults.get('CACHE_GROUPS')) \
+                and self.CACHE_TIMEOUT == self.defaults['CACHE_TIMEOUT']:
```

Comment:
I think it would be best to drop `and self.CACHE_TIMEOUT == self.defaults['CACHE_TIMEOUT']`. If `CACHE_GROUPS` is set at all a warning should be emitted.

Reply:
Hmm, well this check is still necessary, otherwise the old `GROUP_CACHE_TIMEOUT` would override the new setting (if both are set). Is that what you intended?

I can split it out from the check for `DeprecationWarning`.